scripts/sign-macos-pkg.sh

```diff
@@ -65,8 +67,8 @@ function clean_up {
   security list-keychains -s $ORIG_KEYCHAIN_LIST
   security list-keychains
 
-  echo -e "\n### Wiping keychain file..."
-  rm -P "$KEYCHAIN"
+  echo -e "\n### Delete keychain file..."
+  security delete-keychain "$KEYCHAIN"
```

Comment:
this should also happen on errors, I think that is what caused initial issues

Reply:
It does happen on errors. See:
https://github.com/status-im/status-react/pull/7272/files#diff-7020266fb517002c883a982991d1b155R80
```
trap clean_up EXIT
```

Reply:
Although, maybe we should make it to be:
```
trap clean_up EXIT ERR
```
Just in case.